### PR TITLE
GEOMESA-1326 Accumulo 1.7.2 upgrade

### PR DESCRIPTION
**Pull request by jnh5y**
Signed-off-by: Jim Hughes <jnh5y@ccri.com>

### CODE REVIEW

**Comment by jnh5y** [comment]
NB: Don't merge this now.  It is used a WIP and a chance to get Eclipse CQs in.

**Comment by jahhulbert-ccri** [comment]
@jnh5y does 1.7.2 pull in the transitives? some of our deps should change with 1.7.2 like commons-vfs2

**Comment by jnh5y** [comment]
@jahhulbert-ccri good call.  I thought I ran the update-cq shell script.... 

... but we manage commons-vfs2:(. 

More things to fix in PR before mergin.

File: build/cqs.tsv [hunk 1]
```diff
@@ -80,10 +78,9 @@ net.minidev:asm	1.0.2	compile
 net.minidev:json-smart	2.1.1	compile
 net.sf.geographiclib:GeographicLib-Java	1.44	compile
 net.sf.jopt-simple:jopt-simple	3.2	compile
-org.apache.accumulo:accumulo-core	1.6.2	compile
-org.apache.accumulo:accumulo-fate	1.6.2	compile
-org.apache.accumulo:accumulo-start	1.6.2	compile
-org.apache.accumulo:accumulo-trace	1.6.2	compile
+org.apache.accumulo:accumulo-core	1.7.2	compile
+org.apache.accumulo:accumulo-fate	1.7.2	compile
+org.apache.accumulo:accumulo-start	1.7.2	compile
```
Comment (elahrvivaz) [inline build/cqs.tsv]: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=11816

**Comment by jnh5y** [comment]
The CQs for htrace and Accumulo have been approved.

**Comment by cne1x** [comment]
Looks good to me; we have tested it, and it appears to work.

**Comment by jahhulbert-ccri** [comment]
Fails to build with 1.7.2 I will try to dig in soon
```
[error] /home/hulbert/dev/gitsrc/geomesa/geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/EnvironmentCommand.scala:14: object shell is not a member of package org.apache.accumulo.core.util
[error] import org.apache.accumulo.core.util.shell.commands.HelpCommand
[error]                                      ^
[error] one error found
[error] Compile failed at Sep 10, 2016 12:43:53 AM [1.088s]
```

**Pull request closed without merging**